bee-node/src/plugins/dashboard/config.rs

```diff
@@ -74,16 +76,44 @@ impl DashboardConfigBuilder {
     }
 
     pub fn finish(self) -> DashboardConfig {
+        let multi_addr = self
+            .bind_addresses
+            .unwrap_or_else(|| DEFAULT_BIND_ADDRESSES.parse().unwrap());
+        let address = multi_addr
+            .iter()
+            .find_map(|x| match x {
+                Protocol::Dns(std::borrow::Cow::Borrowed("localhost")) => Some(
+                    ("localhost", 0)
+                        .to_socket_addrs()
+                        .expect("error resolving 'localhost'")
+                        .nth(0)
+                        .unwrap()
```

Comment:
pls add a comment why 'unwrap`ing is okay! I suggest:
"Unwrapping here is fine, because 'to_socket-addrs()` didn't return an error, thus we can be sure that the iterator contains at least 1 element."

Reply:
Als here again: I probably should add the same comments to the rest-api config.rs?

Reply:
probably yea :+1: